src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ReportsController.java

```diff
@@ -79,21 +77,19 @@ public ZipFilesSummaryReportListResponse getZipFilesSummary(
         );
     }
 
-    @GetMapping(path = "/zip-files-summary-csv", produces = MediaType.APPLICATION_OCTET_STREAM_VALUE)
+    @GetMapping(path = "/zip-files-summary", produces = "text/csv")
```

Comment:
"text/csv" will just return the comma separated values. I think we want to download the file in csv format